### DISCUSSION TITLE
[Feature] Upsert modified picture when syncing from iOS

### DISCUSSION THREAD

**Discussion by sam-dumont** [immich-app/immich#12714] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, if I modify a picture on iOS (for instance, crop, change the focus distance or length, edit colors, etc) on a picture that was already uploaded to Immich server, the modifications are not synced again. It would be handy to keep all modifications synced to the server (maybe in a stack ?)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6439

**Reply by sam-dumont** [reply]
Sorry! Tried to find an existing discussion but did not use the right search terms it seems...
